src/channel.ts

```diff
@@ -1119,7 +1119,7 @@ export class Channel<StreamChatGenerics extends ExtendableGenerics = DefaultGene
     const key = callbackOrNothing ? (callbackOrString as string) : 'all';
     const valid = isValidEventType(key);
     if (!valid) {
-      throw Error(`Invalid event type ${key}`);
+      console.warn(`Invalid event type ${key}`);
```

Comment:
early return is missing

Reply:
No, we want to continue executing listeners here. The reason being this `throw error` creates blocker for custom events.

Reply:
It's not executing code path but changing listeners so it will put an invalid key

Reply:
I am wondering if we should simply remove this console.warn as well.

Reply:
so do you want to put whatever given by user? makes sense to drop then

Reply:
Yeah!! cc @vanGalilea

Reply:
Ready guys!